services/tasks/local_unix.go

```diff
@@ -25,7 +25,6 @@ import (
 
 var tasksServiceRequires = []plugin.Type{
 	plugins.EventPlugin,
-	plugins.RuntimePlugin,
```

Comment:
was nervous for a sec..  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.gvisor]
      runtime_type = "io.containerd.runsc.v1"  

but then https://github.com/google/gvisor/blob/master/go.mod#L12   

Hopefully everyone is off v1 shim..

Reply:
nit https://github.com/containerd/containerd/blob/main/docs/cri/config.md?plain=1#L76-L77 and https://github.com/containerd/containerd/blob/main/docs/cri/config.md?plain=1#L92-L97 should be removed